src/DssDirectDepositHub.sol

```diff
@@ -163,6 +216,16 @@ contract DssDirectDepositHub {
         emit File(ilk, what, data);
     }
 
+    /**
+        @notice update plan or pool addresses for D3M ilk
+        @dev Requirements:
+        - msg.sender must be authorized
+        - vat is live
+        - the pool is live
```

Comment:
Also here, seems a bit too detailed to me, we don't need to describe all the validations.